src/convertToObject.js

```diff
@@ -3,16 +3,31 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const cssItems = sourceString
+    .split(';')
+    .filter(removedBlankLines => removedBlankLines.includes(':'))
+    .map(item => item.split(':'));
+
+  return cssItems.reduce((value, entries) => {
+    const trimmedProperty = entries[0].trim();
+    const trimmedClass = entries[1].trim();
+
+    return {
+      ...value,
+      [trimmedProperty]: trimmedClass,
```

Comment:
```suggestion
      [trimmedProperty]: trimmedClass,
```
u can trim them here without creating extra variables on each call of callback
also move callback outside of function to avoid creating it every time